lib/es-models/base.js

```diff
@@ -18,6 +19,31 @@ class EsBase {
       return ['b' + this.uri()]
     }
   }
+
+  /**
+   *  All ES models are "jsonable", meaning they can be rendered as a
+   *  plainobject by calling .toJson(). If any sub-models implement toJson,
+   *  they will also be rendered as a plainobject.
+   */
+  async toJson () {
+    // Apply to-json to this object:
+    const plainObject = await toJson(this)
+    // Recursively apply to-json to any child entity that implements toJson:
+    await Promise.all(
+      Object.entries(plainObject).map(async ([key, value]) => {
+        // Handle case where ES model method returns a single ES model:
```

Comment:
assuming that the model methods that would return es models are this.items(), this.holdings(), this.checkincards(), and that other es methods wont, can those be called out somewhere in these comments?

Reply:
Added [this](https://github.com/NYPL/research-catalog-indexer/pull/32/files#diff-75b3e9ac24d4c013fbdacef132adcdf546bd904a9fa0f3010291ac8462c3f450R27-R30)